src/Accelerators/NNPA/Compiler/NNPACompilerUtils.cpp

```diff
@@ -143,9 +111,8 @@ void addPassesNNPA(mlir::OwningOpRef<mlir::ModuleOp> &module,
       emissionTarget = EmitMLIR;
     else {
       pm.addPass(mlir::createCanonicalizerPass());
-      // Add instrumentation for remaining Onnx Ops
-      if (instrumentZHighOps != "" && instrumentZHighOps != "NONE")
-        pm.addNestedPass<FuncOp>(createInstrumentONNXPass());
+      // Add instrumentation for ZHigh Ops
+      pm.addNestedPass<FuncOp>(onnx_mlir::zhigh::createInstrumentZHighPass());
```

Comment:
`onnx_mlir::`   shouldn't be required given we are inside that namespace already.

Reply:
Done. Thanks!